src/Point/Components/PointMachine/PointMachineState.cs

```diff
@@ -1,14 +1,17 @@
 using ReportedPointPosition = EulynxLive.Messages.Baseline4R1.PointPointPositionMessageReportedPointPosition;
+using ReportedDegradedPointPosition = EulynxLive.Messages.Baseline4R1.PointPointPositionMessageReportedDegradedPointPosition;
 using PointMachineStateMessage = EulynxLive.Point.Proto.PointMachineStateMessage.Types;
 
 namespace EulynxLive.Point.Components
 {
     public class PointMachineState
     {
+        public ReportedPointPosition LatestFinalPointPosition { get; set; }
```

Comment:
Why do we need this? we can take the information from DegradedLeft vs DegradedRight